AsaApi/Core/Public/API/ARK/ExtensionsDefinition.h

```diff
@@ -40,7 +40,10 @@ FORCEINLINE FString PlayerControllerExtensions::GetEOSId()
 {
     AShooterPlayerController* _this = static_cast<AShooterPlayerController*>(this);
     FString eos_id = "";
-    _this->GetUniqueNetIdAsString(&eos_id);
+    
+    if (this)
```

Comment:
This check should be entirely unnecessary as "extensions" should exist as member functions and will always have a valid this parameter.
i.e.
`SomePlayerController->GetEOSId()` will **always** crash prior to this check if `SomePlayerController` is null.